frontend/src/component/project/Project/CreateProject/NewCreateProjectForm/CreateProjectDialog.tsx

```diff
@@ -205,17 +273,159 @@ export const CreateProjectDialog = ({
                             }
                         />
                     }
-                >
-                    <Button onClick={onClose}>Cancel</Button>
-                    <CreateButton
-                        name='project'
-                        permission={CREATE_PROJECT}
-                        disabled={
-                            creatingProject || limitReached || loadingLimit
-                        }
-                        data-testid={CREATE_PROJECT_BTN}
-                    />
-                </NewProjectForm>
+                    handleSubmit={handleSubmit}
+                    name={projectName}
+                    setName={setProjectName}
+                    description={projectDesc}
+                    setDescription={setProjectDesc}
+                    errors={errors}
+                    icon={StyledProjectIcon}
+                    onClose={onClose}
+                    testIds={{
```

Comment:
Could the ids be same not configurable? I feel like when testing there is always just 1 formtemplate open, so no confusion.

Reply:
Yeah, maybe? They're used in cypress tests (`frontend/cypress/support/UI.ts`), but cypress should also only have a single form open at the same time. I'll have a look at it.

Reply:
Made some changes now. Let's see whether cypress likes it. Good shout